CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -11,6 +11,10 @@ class X2Helper_Infiltration extends Object config(Infiltration);
 
 var config int PERSONNEL_INFIL;
 var config int PERSONNEL_DETER;
+
+var config int EXFIL_COST_BASEAMOUNT;
+var config int EXFIL_COST_MULTIPLIER;
```

Comment:
I think keeping `INTEL` part in the variable is a good idea, even if makes the variable name quite long